bin/merge_featurecounts.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env/python
+
+import argparse
+import os
+import re
+from collections import defaultdict
+
+
+def do_the_thing(dest_dir,out_file,input_files):
+   table_dict=defaultdict(dict)
+   sample_names=[]
+   gene_list=[]
+   for input_file in input_files:
+       if not gene_list:
+           first_file=True
+       print"Reading from {}".format(input_file)
+       sample_name=os.path.basename(input_file)
+       sample_name= sample_name.replace("Aligned.sortedByCoord.out_gene.featureCounts.txt","")
+       sample_names.append(sample_name)
+       table_dict[sample_name]=dict()
+     
+       #Extract the gene list from the first file
+       if first_file:
+           with open(input_file, 'r') as f:
+               for line in f:
+                   if not line.startswith('E'):
+                       continue
+                   line_info=line.split('\t')
+                   gene=line_info[0]
+                   gene_list.append(gene)
```

Comment:
Personally I'd probably just write this as `gene_list.append(line.split('\t')[0])`